BAZProject/Networking/URLConfiguration.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Paths.swift
+//  BAZProject
+//
+//  Created by Daniel Alberto Vazquez Rodriguez on 03/04/23.
+//
+
+import Foundation
+
+public struct URLConfiguration{
+    var strMethod: String
+    var strHost: String
+    var path: Paths
+    private var strCurrentLocale: String {
+        if #available(iOS 16.0, *){
+            return Locale.current.language.languageCode?.identifier ?? ""
+        }else{
+            return Locale.current.identifier
+        }
+    }
+    
+    public init(strMethod: String = "https", strHost: String = "api.themoviedb.org", path: Paths) {
+        self.strMethod = strMethod
+        self.strHost = strHost
+        self.path = path
+    }
+    
+    public mutating func updateHost(with strHost: String) {
+        self.strHost = strHost
+    }
+    
+    public func configureURL() -> URL? {
+        if strHost.count > 0 {
+            var components = URLComponents()
+            components.scheme = strMethod
+            components.host = strHost
+            components.path = path.rawValue
+            components.queryItems = [URLQueryItem(name: "api_key", value: "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"),
+                                     URLQueryItem(name: "language", value: strCurrentLocale)]
+            
+            return components.url
+        }
+        
+        return nil
```

Comment:
Podemos invertir el case para mejorar la legibilidad, y sugerencia adicional es que es preferible utilizar la propiedad `.isEmpty` en favor de `.count > 0`
```suggestion
        guard !strHost.isEmpty else {
            return nil
        }
        
        var components = URLComponents()
        components.scheme = strMethod
        components.host = strHost
        components.path = path.rawValue
        components.queryItems = [
            URLQueryItem(name: "api_key", value: "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"),
            URLQueryItem(name: "language", value: strCurrentLocale)
        ]
        
        return components.url
```